src/sentry/api/endpoints/integrations/sentry_apps/requests.py

```diff
@@ -46,16 +54,39 @@ def format_request(self, request: Request, sentry_app):
                 # If the org somehow doesn't exist, just don't add it to the result
                 pass
 
+        if org_slug and not formatted_request.get("organization", {}).get("slug") == org_slug:
+            return {}
+
         return formatted_request
 
     def get(self, request: Request, sentry_app) -> Response:
         """
         :qparam string eventType: Optionally specify a specific event type to filter requests
         :qparam bool errorsOnly: If this is true, only return error/warning requests (300-599)
+        :qparam string organizationSlug: Optionally specify an org slug to filter requests
+        :qparam string start: Optionally specify a date to begin at. Format must be YYYY-MM-DD HH:MM:SS
+        :qparam string end: Optionally specify a date to end at. Format must be YYYY-MM-DD HH:MM:SS
         """
+        date_format = "%Y-%m-%d %H:%M:%S"
+        now = datetime.now().strftime(date_format)
+        default_start = "2000-01-01 00:00:00"
+        invalid_date_format_message = "Invalid date format. Format must be YYYY-MM-DD HH:MM:SS."
```

Comment:
You should extract this string from the class so that you can access it in tests.
```py 
INVALID_DATE_FORMAT_MESSAGE = "Invalid date format. Format must be YYYY-MM-DD HH:MM:SS."

class SentryAppRequestsEndpoint(SentryAppBaseEndpoint):
    ...
```